server/cluster/cluster.go

```diff
@@ -41,15 +42,24 @@ type Cluster struct {
 	shardIDAlloc  id.Allocator
 }
 
-func (c *Cluster) GetNodesSize() int {
+func (c *Cluster) GetNodes() []*Node {
 	c.lock.RLock()
 	defer c.lock.RUnlock()
-	return len(c.nodesCache)
+
+	nodes := make([]*Node, 0, len(c.nodesCache))
+	for _, node := range c.nodesCache {
+		nodes = append(nodes, &Node{
+			meta:     ConvertNodeTOPB(node),
```

Comment:
The method name `ConvertNodeTOPB` maybe should be renamed to `ConvertNodeToPB`.

BTW, why isn't the returned type `Node` the same as the one in the `c.nodesCache`?